ios/Sources/FavoritesFeature/FavoritesScreen.swift

```diff
@@ -1,53 +1,67 @@
 import ComposableArchitecture
 import Component
 import Introspect
+import SettingFeature
 import SwiftUI
 import Styleguide
 
 public struct FavoritesScreen: View {
     private let store: Store<FavoritesState, FavoritesAction>
+    @ObservedObject private var viewStore: ViewStore<FavoritesState, FavoritesAction>
```

Comment:
Why did you define this property?

Are there any problem with using `WithViewStore` ?

Reply:
I felt uncomfortable defining two times WithViewStore, but I noticed that I can define only one.
I'll remove this.   　

Reply:
DONE